.github/workflows/ci.yaml

```diff
@@ -5,13 +5,15 @@ on:
   pull_request:
     branches:
       - main
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Do we want to run the CI daily, isn't weekly enough?

Reply:
I think daily is good.